common/threadloop.hpp

```diff
@@ -102,10 +102,11 @@ class threadloop : public plugin {
 				break;
 			}
 			case skip_option::stop:
-				stop();
-				break;
+				goto break_loop;
```

Comment:
`break` would break out of the `switch`. I want to break out of the loop which encloses the `switch` (like a `break` and then `break`). `goto` is the best way to do that in C/C++ according to [StackOverflow](https://stackoverflow.com/questions/27788326/breaking-out-of-nested-loop-c).

Reply:
If we don't anticipate post-loop code blocks, we could also return directly from the switch case. Neither solution is pretty.